### PR TITLE
Adding biome.package.json

### PR DESCRIPTION
**Pull request by CarlosGamero**
## Changes

Please describe

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/dev/biome-config/configs/biome-package.jsonc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../../../node_modules/@biomejs/biome/configuration_schema.json",
+  "extends": ["./biome-base.jsonc"],
```
Comment (kibertoad) [inline packages/dev/biome-config/configs/biome-package.jsonc:3]: unfortunately, extending from the same package is not supported by biome, it needs to be combined on a consumer side

File: packages/dev/biome-config/configs/biome-package.jsonc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (kibertoad) [inline packages/dev/biome-config/configs/biome-package.jsonc:1]: can you mention new preset in the documentation?

File: packages/dev/biome-config/configs/biome-package.jsonc (hunk 1 above)
Comment (CarlosGamero) [reply packages/dev/biome-config/configs/biome-package.jsonc:3]: I was experiencing it right now 😓 then I guess that I should remove this remove on on package do 
```
extends: [base, package]
```
right?

File: packages/dev/biome-config/configs/biome-package.jsonc (hunk 2 above)
Comment (CarlosGamero) [reply packages/dev/biome-config/configs/biome-package.jsonc:1]: On it, thank you!

Comment (CarlosGamero) [reply packages/dev/biome-config/configs/biome-package.jsonc:1]: btw, is the following correct:
```Backend:

```json
{
	"$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
	"extends": ["./node_modules/@lokalise/biome-config/configs/biome-base.jsonc", "./node_modules/@lokalise/biome-config/configs/biome-esm.jsonc"]
}
```

I think we are not using esm config, right? asking just to understand it 🙏

Comment (kibertoad) [reply packages/dev/biome-config/configs/biome-package.jsonc:1]: newer services (Billing, Workflows) use ESM, Autopilot doesn't (benefits of a switch do not currently justify the effort needed)

File: packages/dev/biome-config/configs/biome-package.jsonc (hunk 1 above)
Comment (kibertoad) [reply packages/dev/biome-config/configs/biome-package.jsonc:3]: yup!

**Review by kibertoad — APPROVED** [review]

**Comment by CarlosGamero** [comment]
Background-jobs-common had lint issues, they are not important and I don't think we need to release a new version of it so marking PR as `skip-release` and I will release new `biome-config` version manually

**Pull request merged**